original commit

n2ic opened the pull request:
Alex - Finally tackled GitHub to fork my changes to MorseRunner. The most significant change is the port to the Lazarus IDE. Since you use Delphi, you probably need to be careful about what you might merge. Functionally, these changes support multiple instances of MR, each outputting on their own sound card. I'll send you e-mail with more info.

n2ic committed:
original commit

BH1SCW commented:
BG4FQD and me (BH1SCW)   started another branch of MR few days ago, because original version can't upload score anymore and no more new feature is developed. so we set up another hight score page and add some feature to it, for example changing to Unicode, add some nice font, we hope some guys can continue to patching and submitting to MR, we love this software and it's greate for helping for learing morse code and parctising contest.

73 de BH1SCW

VE3NEA commented:
:+1:

n2ic committed:
added udp server

f6fvy committed:
Add callers to the pile-up when a QSO is done

Formally, no caller was added to a QSO was done, and hitting F1 several times in a row was adding too many callers regarding the activity setting.

With this mod:

1/ New callers may be added once a QSO is entered.
2/ The total number of callers is always respecting the activity setting

n2ic committed:
Merge pull request #1 from f6fvy/patch-1

Add callers to the pile-up when a QSO is done

n2ic committed:
Add files via upload

n2ic committed:
Add files via upload

n2ic committed:
Delete SndOut.pas

n2ic committed:
Add files via upload

n2ic committed:
Merge pull request #2 from n2ic/mr042620

Mr042620

VE3NEA commented:
Thank you for working on improving Morse Runner. I no longer maintain this project, so please do not merge your code upstream, release your own versions of the program instead.

n2ic commented:
Hi Alex,

Yes, I screwed up with github. Sorry about that !

73,
Steve, N2IC

On 04/26/2020 07:13 PM, Alex Shovkoplyas wrote:
> Thank you for working on improving Morse Runner. I no longer maintain this 
> project, so please do not merge your code upstream, release your own versions of 
> the program instead.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/VE3NEA/MorseRunner/pull/3#issuecomment-619657929>, or 
> unsubscribe 
> <https://github.com/notifications/unsubscribe-auth/AD45JRAPFQ6PQBRB6ANEUXDROTL4LANCNFSM4CKRRTKQ>.
>

n2ic committed:
Add files via upload

n2ic committed:
Merge pull request #3 from n2ic/May-8-2020

Add files via upload

n2ic committed:
Add files via upload

n2ic committed:
Add files via upload

n2ic committed:
Merge pull request #4 from n2ic/May8otherdirectories

May8otherdirectories

n2ic committed:
Add files via upload

n2ic committed:
Merge pull request #5 from n2ic/may102020

Add files via upload

n2ic committed:
Add files via upload

n2ic committed:
Merge pull request #6 from n2ic/May102020B

Add files via upload

n2ic committed:
Add files via upload

n2ic committed:
Merge pull request #7 from n2ic/may102020C

Add files via upload

n2ic committed:
Add files via upload

n2ic committed:
Merge pull request #8 from n2ic/temp

Add files via upload

Commit:
Sync Ini.pas with development area

Commit:
Sync with development area

Commit:
Fixed time out in Station.pas and added call database

Commit:
Changed FileExists32 calls to FileExist
